sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_customer_statsbeat_types.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+from enum import Enum
+from typing import Dict, Any, List, Optional, TypedDict
+
+# Language constant for customer statsbeat
+STATSBEAT_LANGUAGE = "python"
+class TelemetryType(str, Enum):
+    AVAILABILITY = "AVAILABILITY"
```

Comment:
These types are not just specific to statsbeat so you can probably put them under `_constants.py`